sk.c

```diff
@@ -304,8 +304,11 @@ int sk_receive(int fd, void *buf, int buflen,
 	struct cmsghdr *cm;
 	struct iovec iov = { buf, buflen };
 	struct msghdr msg;
-	struct timespec *sw, *ts = NULL;
+	struct timespec *sw;
+	struct timespec *ts = NULL;
```

Comment:
Probably best to leave these two lines as-is, or split out to a separate patch with a justification for why it should be changed.